Add competition & Tech day reminder email (#28)
srobo/volunteer-emails: SR2020/2020-02-20-competition-reminder.md

---
to: All Volunteers
subject: We need competition volunteers!
---

Hello all,

Quick update on the volunteering opportunities:

> ## We still need volunteers for the competition!

Reviewer: markdown nit: newlines before headings.
```suggestion

## We still need volunteers for the competition!
```

## We still need volunteers for the competition!

> The dates will be April 18th - 19th, with set-up days on the 16th and 17th if you’re available! The venue is RUSU, the University of Reading Student Union.

Reviewer 2: Maybe worth linking to the event page on the website? It's fairly empty now, but will eventually contain more than enough information!

Author: Done!

> Accommodation will be free for volunteers, and we’ll provide lunch on the competition days. We may also reimburse travel depending on the budget, so keep your train ticket receipts!

Reviewer: Some tweaks:
```suggestion
Accommodation will be provided for volunteers, and we’ll provide lunch on the competition days. We may also reimburse travel depending on the budget, so keep your travel receipts!
```

- "provided" over "free" clarifies that we've also done the organising
- "travel" over "train tickets" accounts for other modes of travel

Please sign up if you fancy coming: https://forms.gle/foHy82uY98wiPm587

**The deadline for applying to ensure you get accommodation is March 14th!!**

> ## Helping with Tech Days

Reviewer: markdown nit: newlines after headings
```suggestion
## Helping with Tech Days

```

> Tech Days are opportunities for teams to spend a whole day working on their robot with lots of help available from us. They’re also an opportunity to see how other teams are doing or get more direct help with your robots.

Reviewer: ```suggestion
Tech Days are opportunities for teams to spend a whole day working on their robot with lots of help available from us. They’re also an opportunity to meet and get to know other volunteers.
```

> We provide a space for you to work in, with power and internet access, as well as volunteers able to help you with your kits and hands-on guidance with your robots.

Reviewer: Maybe just drop this? Or replace it with something about what a volunteer should bring (i.e: just themselves).

The next tech days coming up will be on:

| Date | Location | 
|February 29th |[London (Thread, Whitechapel)](https://studentrobotics.org/events/sr2020/london-tech-day-february/) |
| March 7th | [University of Southampton](https://studentrobotics.org/events/sr2020/southampton-tech-day-march/) |
| March 28th | [London (Thread, Whitechapel)](https://studentrobotics.org/events/sr2020/london-tech-day-march/) |

> If you’re free to help out on a tech day, then just turn up! The events pages have all the details. Don’t worry if you’re new, we’d be happy to help you out with assisting teams or doing other things for the competition!

Reviewer 2: There's actually a signup form in drive for volunteers for tech days.
